rasa/shared/utils/validation.py

```diff
@@ -246,19 +246,20 @@ def validate_training_data_format_version(
         if isinstance(parsed_version, LegacyVersion):
             raise TypeError
 
+        if str(version_value) < LATEST_TRAINING_DATA_FORMAT_VERSION:
+            rasa.shared.utils.io.raise_warning(
+                f"Training data file {filename} has a lower "
+                f"format version than your Rasa Open Source installation: "
+                f"{version_value} < {LATEST_TRAINING_DATA_FORMAT_VERSION}. "
+                f"Rasa Open Source will read the file as a version "
+                f"{LATEST_TRAINING_DATA_FORMAT_VERSION} file. "
+                f"Please update your version key to "
+                f"{LATEST_TRAINING_DATA_FORMAT_VERSION}. "
+                f"See {DOCS_URL_TRAINING_DATA}."
```

Comment:
Just noticed this docs link reference, could you have a look to see if we need to insert a comment that the training data version in `3.0` should be updated to `3.0`?

Reply:
in that docs page, I mean, sorry keep pressing enter without finishing my comment 😅